CONTRIBUTING.md

```diff
@@ -55,7 +55,6 @@ The following list of instructions pertains to commonplace site updates by contr
   1. Use only the theme colors available at [themeStyles.js](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js) instead of external color codes.
   2. Avoid adding `rel="noreferrer"` attribute to internal links on the site.
 
-
 ## Adding Images and Icons
```

Comment:
Please note that the site has a light mode and a dark mode theme. Please see theme.md.